public void test3() throws AssertionError {
    assert_true(1 == 2, "1 == 2");
}//test1_run


